account_payment_order/__init__.py

```diff
@@ -0,0 +1,5 @@
+# -*- coding: utf-8 -*-
+# (c) 2015 brain-tec AG (http://www.braintec-group.com)
```

Comment:
You should put here all the companies or not put anyone (best option, as they are too much). But in main files you should include all of them with their respective dates.